### ISSUE TITLE
Compilation issue

### ISSUE THREAD

**Issue by martinos** [balena/elixir-sippet#4]
When I try to compile an app that uses the library I get the following error

```
could not compile dependency :sippet, "mix compile" failed. You can recompile this dependency with "mix deps.compile sippet", update it with "mix deps.update sippet" or clean it with "mix deps.clean sippet"
** (ArgumentError) all arguments for System.cmd/3 must be binaries
    (elixir) lib/system.ex:625: System.cmd/3
    /Users/mchabot/dev/ispt/test_sip/deps/sippet/mix.exs:85: Mix.Tasks.Compile.Make.run/1
```

I am on Elixir 1.8

**Comment by balena** [comment]
I just published the version 0.6.1 on Hex. Please double check if it works for you now.

**Comment by martinos** [comment]
Thanks, it works.
